element_lab/lab.py

```diff
@@ -24,12 +25,21 @@ def activate(schema_name: str, create_schema: bool = True, create_tables: bool =
 
 @schema
 class Organization(dj.Manual):
+    """Top-level list of all organizations involved in any of the projects.
+
+    Attributes:
+        organization         ( varchar(24)   ): Abbreviated organization name.
+        organization_name    ( varchar(255)  ): Full organization name.
+        organization_address ( varchar(512)  ): Address of the organization.
+        organization_comment ( varchar(1024) ): Additional notes on the organization.
```

Comment:
I think mkdocs will render the data type literally on the [api page](https://datajoint.com/docs/elements/element-lab/0.1/api/element_lab/lab/) as a table, so the padding would look odd

```suggestion
        organization ( varchar(24) ): Abbreviated organization name.
        organization_name ( varchar(255) ): Full organization name.
        organization_address ( varchar(512) ): Address of the organization.
        organization_comment ( varchar(1024) ): Additional notes on the organization.
```

Reply:
Yeah I wasn't actually sure how that would be rendered.